examples/anchors.yaml

```diff
@@ -31,7 +31,7 @@ meter_provider:
             # expand the otlp-exporter anchor and add metric specific configuration
             <<: *otlp-exporter
             temporality_preference: delta
-            default_histogram_aggregation: exponential_bucket_histogram
+            default_histogram_aggregation: base2_exponential_bucket_histogram
```

Comment:
This reflects the naming in the spec: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk_exporters/otlp.md